documentation/use-cases/Story4-UseCaseNarrative.md

```diff
@@ -0,0 +1,54 @@
+# STORY 4 - USE CASE NARRATIVE
+
+**Use case**: Enroll students  
+
+**Scope**:  /   
+
+**Level**: User-goal  
+
+**Intention in context**: The administrative officer wants to enroll students in the school  
+
+**Primary actor**: Administrative officer
+
+**Secondary actor(s)**:  /
+
+**Stakeholders' interests**: 
+
+* Administrative officer: would like to enroll students in a small number of steps
+* Principal: would like to offer school services (e.g. class forming, rooms handling, etc.) on the basis of the number of students enrolled
+* Parent: would like his/her child/children to be successfully enrolled in the school
+
+**Precondition**:
+
+* The maximum number of students enrolled should be greater than the actual one
```

Comment:
Qua cambierei la frase in "The maximum number of enrollable students should be greater than the currently enrolled one", giusto per essere un po' più chiari

Reply:
Ovviamente, intendevo quello